src/main/mdo/assembly-component.mdo

```diff
@@ -568,6 +568,15 @@
           build should be included in this dependency set. Default value is false. (Since 2.2)
           ]]></description>
         </field>
+        <field>
+          <name>useOptionalDependencies</name>
+          <version>2.1.0+</version>
+          <type>boolean</type>
+          <defaultValue>false</defaultValue>
```

Comment:
I wonder if the defallt here should be true to avoid changing existing behavior.